apps/opik-backend/src/main/resources/liquibase/db-app-state/migrations/000011_add_automation_rule_evaluators_type_user_defined_metric_python.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+--changeset andrescrz:000011_add_automation_rule_evaluators_type_user_defined_metric_python
+
+ALTER TABLE automation_rule_evaluators MODIFY COLUMN `type` ENUM('llm_as_judge', 'user_defined_metric_python') NOT NULL;
```

Comment:
Just be careful because once the first row with `user_defined_metric_python` is inserted, we cannot run the rollback